docs/src/monitoring.md

```diff
@@ -359,7 +359,7 @@ go_threads 18
 ### User defined metrics
 
 This feature is currently in *beta* state and the format is inspired by the
-[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/master/queries.yaml) <!-- wokeignore:rule=master -->
+[queries.yaml file](https://github.com/prometheus-community/postgres_exporter/blob/v0.12.1/queries.yaml)
```

Comment:
Um. Perhaps worth adding a note like `(deprecated)` or an explicit mention of the version?